uber/utils.py

```diff
@@ -77,6 +77,30 @@ def localize_datetime(dt):
     return dt.replace(tzinfo=UTC).astimezone(c.EVENT_TIMEZONE)
 
 
+def ceil_datetime(dt, delta):
```

Comment:
This doesn't seem to actually be used, either here or in the companion panels PR.

Reply:
Yeah, another utility function that was used at one point, but is no longer used.